### DISCUSSION TITLE
Lower `update_ms` than `100ms`?

### DISCUSSION THREAD

**Discussion by kolpav** [aristocratos/btop#421] [General]
Is it possible to force refresh rate lower than 100ms? I have 0 uses for it I would just like to try it

**Comment by aristocratos** [comment]
@kolpav 
Sure, change `i_value < 100` to `i_value < 1` in:
https://github.com/aristocratos/btop/blob/ee0945969d46c17a291142b600173a11068e390d/src/btop_config.cpp#L385-L389

And compile.
However I would recommend you first run `btop --debug` and check that the timer for `total` in the debug box is below what you wanna use. The timer is in microseconds so 100'000 µs →  100 ms. If you set too low update value, keyboard input might become unresponsive.

There is also the possibility that some of the values collected from the kernel isn't updated internally faster than the value you set, in which case you would just get the same values re-reported.

So the default minimum of 100 ms is there because below that number the accuracy of the reported information worsen and input might become unresponsive on slower systems.

**Reply by kolpav** [reply]
got it 1ms refresh rate it is then, wish me luck
